### ISSUE TITLE
How to use appsettings.json to pre-config wikitraccs.gui?

### ISSUE THREAD

**Issue by PretoPlasma** [WikiTransformationProject/wikitraccs-releases#40]
Hi,
we plan to run wikitraccs.gui controlled by rpa robot.

However the appsettings.json to pre-config wikitraccs.gui seems to be ignored when starting the gui.

According to [WikiTraccs Documentation](https://www.wikitransformationproject.com/docs/reference/settings/) it should be enough to place the file inside the same folder of the gui. However, as I wrote, it is not recognized.

Is there anything else to do to get the tool fetching the config from this file?

We have tried it with wikitraccs 0.2.0 and 0.2.3 @win_x64.

Looking forward to a quick reply.
Best regards

**Comment by PretoPlasma** [comment]
Thank you very much for your information. It took a while to prepare my test environment for automated execution.
I have used your recommendations on https://www.wikitransformationproject.com/docs/reference/settings/#configuring-via-configuration-file.
However when trying to automate the execution of the wikitraccs.console I found out that the console is not closing when all done. How can I find out that the migration is finished?
